src/components/SignUpForm.js

```diff
@@ -1,8 +1,43 @@
 import React, { Component } from 'react';
-import { Form, Message } from 'semantic-ui-react';
+import { Form, Message, Checkbox, Input, Label, Button} from 'semantic-ui-react';
 import { validate } from 'email-validator';
 
+import { validateZipCode, validateWage } from '~/util';
+
+import EditableList from './EditableList';
+
 class SignUpForm extends Component {
+
+  constructor(props) {
```

Comment:
You don't need to use a constructor to create instance variables.  instead simply place assignments in the first class block scope. 

Check out any of my other Components to see what that looks like.